doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1533,18 +1581,8 @@ Response code:
             ],
             "scopes": [
                 {
-                    "type": "FIWARE::Filter::qString",
-                    "value": "temperature>40;colour=black,red;+status"
-                },
-                {
-                    "type": "FIWARE::Filter::Location",
-                    "value": {
-                        "circle": {
-                            "centerLatitude": "40.418889",
-                            "centerLongitude": "-3.691944",
-                            "radius": "13500"
-                        }
-                    }
+                    "type": "FIWARE::...",
```

Comment:
The specific scopes to take into account in NGSIv2 (i.e. list with specifid type such as "FIWARE::StringQuery", etc) has not been "consolidated" yet. Thus, maybe a good idea is to leave this with TBD, e.g.:

 ```
{
   "type": "FIWARE::TBD",
   "value": "TBF"
}
```